### DISCUSSION TITLE
Preparing Youth to Thrive: Promising Practices for Social and Emotional Learning (2016)

### DISCUSSION THREAD

**Discussion by catherinet1** [life-itself/community#850] [Research]
Notes on: Smith, Charles, Gina McGovern, Reed Larson, Barbara Hillaker and Stephen Peck, *Preparing Youth to Thrive: Promising Practices for Social and Emotional Learning* (2016), Forum for Youth Investment, Washington D.C.

# The project

- The aim of this project was to 'identify promising practices for building skills in six areas: emotion management, empathy, teamwork, initiative, responsibility, and problem solving'
- The project looked at 8 exemplary organisations offering OST (out-of-school-time) youth programmes. All programmes combined work on a concrete project, e.g. creating a community service or building a boat, with SEL

# The guide

- This guide presents:
	- 32 standards of SEL practice in 6 domains
	- 58 indicators of SEL practice in 6 domains
	- 4 curriculum features shown to be foundational for SEL practice

# SEL Domains

- Domain = a category of SEL practice and skill
- Selected based on prior research, input of project partners, funder priorities, and desire to talk about SEL in plain language

![Screenshot 2023-10-05 at 18 35 06](https://github.com/life-itself/community/assets/97066487/39b9b157-8218-474f-bef5-4033ebfe9e69)

--> These seem pretty similar to IDG categories of skills/capacities.
- Those are:
	- Being 
	- Thinking
	- Relating
	- Collaborating 
	- Acting

# Key Curriculum Features

- The following features are identified as key to supporting SEL
	- Content Sequence (Project and SEL)
	- Safe Space
	- Responsive Practice
	- Supports for Staff

--> Key is that "curriculum" as they see it not only includes content and sequence of delivery but also *supports for staff* to implement the programme (including things like supporting staff personal/professional development, energy levels, and skill, and appropriate recruitment of participants)

![Screenshot 2023-10-06 at 09 54 00](https://github.com/life-itself/community/assets/97066487/27759cc0-4303-4d68-974b-ac3acb762a84)

## SEL Content Sequence

Progression through the domains of SEL
- The research found a pattern in the progression of SEL:
	- 'We found the SEL Challenge programs consistently began with exposure to empathy and teamwork experiences, whereas more intensive experiences with initiative and responsibility occur later in the offering design as the project content sequence picks up steam' (i.e. as the level of complexity of project tasks increases) (p.21)
	- 'Staff practices and youth experience in the emotion management and problem solving domains occur throughout the offering'
- This progression through SEL domains is framed also in terms of progression from learning about self, then others, then wider world (p.21)

Importance of community engagement
- In most of the programmes, there was also a progression from being mostly in the more private space of the programme towards more public settings with more engagement with wider community. This supports youth to: demonstrate their skills, receive feedback and validation from their community, and thus see their own value and potential for impact in their community

Agency as the super skill
- ''"Social and emotional skills are action skills." We heard this so many times in the SEL Challenge -- from both staff and youth -- that we came to refer to agency as the "super skill."' When youth demonstrate mastery of social and emotional skills, they are successfully regulating their emotions, attention, motivation, and behavior to engage with, or exit, the contexts they find themselves in. Youth who learn to self-regulate are more successful at doing the things they want to do during the offering, as well as in other facets of life. The concept of agency also refers to the experience of control in their own lives that youth experience when they are trusted to make decisions about things that affect them.' (p.22)

# What do they mean by 'practices'?

- Their definition of staff practices includes:
	- staff behaviours, e.g. modelling appropriate use of emotion
	- program structures that the staff put into place, e.g. recruitment policy
- These two things shape 'key youth experiences' 

--> I think they make an important here pointing out staff behaviour as a key practice and contributor to youth experience, as well as whatever content or structure might make up the programme

# Standards for SEL Practice

- All of the 8 programs studied combined SEL with a concrete work project, e.g. creating a community service project or building a boat
	- --> I am curious about the role/function of the concrete work or service project. How well does SEL work without this? Seems like an important component for linking personal transformation to systemic transformation, potentially? What difference does it make what kind of project it is? E.g. whether it's clearly community service etc.?

![Screenshot 2023-10-06 at 09 49 24](https://github.com/life-itself/community/assets/97066487/f50256d5-5e3f-49d2-ad48-d010eb0bdc37)
![Screenshot 2023-10-06 at 09 49 35](https://github.com/life-itself/community/assets/97066487/66174984-47d2-4833-8967-6f2d8b543956)

# Notes

- Other terms used in the field to refer to social and emotional skills:
	- 21st century skills
	- soft skills
	- character education
	- experiential learning
	- positive youth development

**Comment by catherinet1** [comment]
Hi @matthewmccarthy11 and @BoazBF, I posted my notes on one of the articles (cf https://github.com/life-itself/labs/issues/173) here, so that you can start with this summary if you want, and so that we can share notes and comments here if we want.

welcoming reactions and questions from others too.

**Comment by matthewmccarthy11** [comment]
Really interesting how they do correspond with the IDG outline to some degree. I would also point out that the "features identified as key to supporting SEL" are very similar as well to the creative community model, which are: safety, intentional challenge, and youth adult partnerships. 

Although I am not exactly sure how to connect the dots in relation to our project in a meaningful way, other than saying that 'these frameworks are similar'. There could be an interesting way to do that, but I'm not sure what it is. 

Another question: how do we consider 'SEL' more broadly? Thinking here about whether SEL is an 'ontological framework' in relation to this post: https://github.com/orgs/life-itself/discussions/859

Thanks again for putting these notes together, super helpful @catherinet1
